src/js/themes/base.js

```diff
@@ -364,6 +364,14 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         radius: `${baseSpacing * 0.75}px`,
       },
       // color: { dark: undefined, light: undefined }
+      normalized: {
```

Comment:
In order to allow more theme flexibility, I think we should allow the theme to control background and border for all flavors of icon. So: `theme.button[flavor].background`, `theme.button[flavor].color`, `theme.button[flavor].border`. And then per-flavor: `theme.button[flavor].active.background`, `theme.button[flavor].hover.background`, etc. and flavor-agnostic state variations: `theme.button.disabled.background = "opacity: 0.7"`.

Reply:
Great, thanks. Let me dive into this more.